packages/components/src/Icons/custom/ExpandMoreRounded.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import createSvgIcon from "../../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <path d="M15.88 9.29001L12 13.17L8.11999 9.29001C7.72999 8.90001 7.09999 8.90001 6.70999 9.29001C6.31999 9.68001 6.31999 10.31 6.70999 10.7L11.3 15.29C11.69 15.68 12.32 15.68 12.71 15.29L17.3 10.7C17.69 10.31 17.69 9.68001 17.3 9.29001C16.91 8.91001 16.27 8.90001 15.88 9.29001V9.29001Z" />,
```

Comment:
I got this path from the UI kit: https://www.figma.com/file/YPTOEVRuas5ytXQEcwm4FK/UI-Kit-Sticker-Sheet-Core?node-id=649%3A2126
Here's the blocky version in `traject`: https://github.com/FB-PLP/traject/blob/77d303699345622a3f8e0aebe013fbf3c1a7d421/app/assets/javascripts/v2/core/EDSCandidates/SvgIcon/ExpandMore.jsx